Server/JokesOnYou.Web.Api/Controllers/JokesController.cs

```diff
@@ -29,5 +29,12 @@ public async Task<ActionResult<IEnumerable<JokeReplyDto>>> GetAllJokesAsync()
             return Ok(jokeDtos);
         }
 
+        [HttpDelete("{id}")]
+        public async Task<ActionResult<IEnumerable<JokeReplyDto>>> DeleteJoke(int id)
```

Comment:
We are not returning anything rather than NoContent()

Reply:
Forgot to remove it, whoops